### PR TITLE
Fix Shape Divider default applied classes

### PR DESCRIPTION
**Pull request by AnthonyLedesma**
### Description

Improper PHP logic resulted in improperly applying class names to shape divider blocks on the published pages. 
Fixed logic and added a new E2E test that would detect this condition in the future.

### Screenshots

Before
Screenshot 2023-08-10 at 9 06 28 AM

After
Screenshot 2023-08-10 at 9 05 41 AM

### Types of changes

PHP Logic.
Testing.

### How has this been tested?

Manually, E2E.

### Acceptance criteria

Should properly apply default classes when expected. 

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/shape-divider/index.php [hunk 1]
```diff
@@ -51,11 +51,11 @@ function coblocks_render_coblocks_shape_divider_block( $attributes ) {
 		? "background-color: {$attributes['customBackgroundColor']};"
 		: false;
 
-	$is_vertically_flipped = isset( $attributes['verticalFlip'] )
+	$is_vertically_flipped = ( isset( $attributes['verticalFlip'] ) && true === $attributes['verticalFlip'] )
```
Comment (jrtashjian) [inline src/blocks/shape-divider/index.php:54]: We can simplify this with `! empty( $attributes['verticalFlip'] )`

Alternatively, we can swap the condition results and remove the negation:
```
empty( $attributes['verticalFlip'] ) ? false : 'is-vertically-flipped'
```

**Review by jrtashjian — APPROVED** [review]

File: src/blocks/shape-divider/index.php (hunk 1 above)
Comment (AnthonyLedesma) [reply src/blocks/shape-divider/index.php]: Great suggestion.

**Comment by AnthonyLedesma** [comment]
Trying to work with GitHub Actions issues all day with this PR. Unfortunately, GitHub is experiencing some issues with actions today, and our release pipeline relies on GitHub Actions.

**Comment by AnthonyLedesma** [comment]
Tests are passing locally. Merging without CICD passing.

**Pull request merged**